README.md

```diff
@@ -222,18 +240,64 @@ Algorand smart contract debugging process uses `tealdbg` command line of algod i
 
 **Note**: Always use `tealdbg` with `--listen 0.0.0.0` or `--listen [IP ADDRESS]` flags, if access is needed to tealdbg from outside of algod docker container!
 
-#### tealdbg examples:
+### tealdbg examples
 
 Debugging smart contract with Chrome Developer Tools (CDT):
-```~$ ./sandbox tealdbg debug ${TEAL_PROGRAM} -f cdt -d dryrun.json```
+`~$ ./sandbox tealdbg debug ${TEAL_PROGRAM} -f cdt -d dryrun.json`
 
 Debugging smart contract with Web Interface (primal web UI)
-```~$ ./sandbox tealdbg debug ${TEAL_PROGRAM} -f web -d dryrun.json```
-  
+`~$ ./sandbox tealdbg debug ${TEAL_PROGRAM} -f web -d dryrun.json`
+
 The debugging endpoint port (default 9392) is forwarded directly to the host machine and can be used directly by Chrome Dev Tools for debugging Algorand TEAL smart comtracts (Goto url chrome://inspect/ and configure port 9392 before using please).
 
 Note: If a different port is needed than the default, it may be changed by running `tealdbg --port YOUR_PORT` then modifying the docker-compose.yml file and change all occurances of mapped 9392 port with the desired one.
 
-## Errors
+## Sandbox Interactive Debugging with VSCode's `Remote - Container` Extension
+
+Here are steps you can take if you want to run an interactive debugger with an `indexer` running on the sandbox. Analogous instructions work for `algod` as well.
```

Comment:
Can you please mention _why_ someone would want to do this and that its probably a more advanced thing to do

Reply:
👍